data/transport missions.txt

```diff
@@ -1264,15 +1264,15 @@ mission "Drug Running 1"
 		illegal 40000
 	to offer
 		random < 10
-		"said no to drugs" < 4
+		"rejected illegal jobs" < 4
```

Comment:
As mentioned above, this change isn't needed - being able to differentiate between flavors of illegality is a part of character building